### DISCUSSION TITLE
[dnd-kit example] List jumps when dragging near the edge of the list

### DISCUSSION THREAD

**Discussion by reifocS** [inokawa/virtua#588] [Q&A] [answered]
[Screencast from 10.12.2024 12:51:24.webm](https://github.com/user-attachments/assets/6273bfb3-3ded-40e8-9eb2-73fc38237b9d)

**Describe the bug**
Scroll position resets when dragging near the end of the list, creating a bouncing effect

**To Reproduce**
I created a minimum reproducible example here
https://codesandbox.io/p/sandbox/wqsrdk

**Expected behavior**
List jumps around when scrolling near the edge of the list

**Platform:**

- OS: [Linux]
- Browser: [Chrome]
- Version of this package: [0.39.0]
- Version of framework: [react, vue, solid-js, svelte] [x.x.x]

**Additional context**
It only happens when the count of item is below a certain threshold

**Comment by inokawa** [comment]
It may be a similar issue to #417

**Reply by inokawa** [reply]
Related issues:
- https://github.com/inokawa/virtua/issues/604
- https://github.com/clauderic/dnd-kit/discussions/1372#discussioncomment-12887917

**Reply by inokawa** [reply, marked as answer]
https://github.com/inokawa/virtua/issues/604#issuecomment-3956889150

**Comment by gaburielcasado** [comment]
On drag start, I save the index of the item being dragged, and conditionally pass an array with that index to `keepMounted`. If no item is being dragged, then I just pass an empty array to `keepMounted`. Haven't seen the list jumping anymore since doing that. Not a solution obviously, but seems to be a stable workaround.
